app/controllers/dashboard_controller.rb

```diff
@@ -0,0 +1,8 @@
+class DashboardController < ApplicationController
+
+  def index
+    @payload = Job.payload
```

Comment:
i encourage you to maybe choose a better method name here? it's not easy to understand what the value of `@payload` is from here